lib/Page.js

```diff
@@ -165,21 +165,50 @@ Page.prototype.prepareTemplateData = function () {
 };
 
 /**
- * Records h1,h2,h3 headings into this.headings
- * @param renderedPage a page with its headings rendered
+ * Records headings in rendered page into this.headings
  */
-Page.prototype.collectHeadings = function (renderedPage) {
-  const $ = cheerio.load(renderedPage);
+Page.prototype.collectHeadings = function () {
+  this.collectHeadingsInContent(fs.readFileSync(this.resultPath));
+};
+
+/**
+ * Records headings inside content into this.headings
+ * @param content that needs to collect headings
+ */
+Page.prototype.collectHeadingsInContent = function (content) {
+  let $ = cheerio.load(content);
+  $('panel').each((index, panel) => {
+    if (panel.attribs.header) {
+      this.collectHeadingsInContent(md.render(panel.attribs.header));
+    }
+  }).filter((index, panel) => panel.attribs.expanded !== undefined)
+    .each((index, panel) => {
+      if (panel.attribs.src) {
+        const includePath = path.resolve(this.resultPath,
+                                         path.relative(this.sourcePath,
+                                                       path.relative(this.baseUrl, panel.attribs.src)));
```

Comment:
There seems to be some implementation error computing the include path. I get this error:

Console:
```
warning: Error: ENOENT: no such file or directory, open 'C:\Users\other._include_.html'
warning: Error: ENOENT: no such file or directory, open 'C:\Users\other._include_.html'
error: ENOENT: no such file or directory, open 'C:\Users\other._include_.html'
```

`index.md`:
```html
<panel header="Included panel" src="other.md" expanded>
</panel>
```

`other.md`:
```markdown
# Other Heading

Other heading
```